odoo/addons/base/models/res_partner.py

```diff
@@ -33,9 +33,32 @@
 def _lang_get(self):
     return self.env['res.lang'].get_installed()
 
+def timezone_offset(tz):
+    """Take the timezone in a string format and return the hour offset.
+
+    e.g.
+        'Africa/Cairo' -> '+0200'
+    """
+    utcnow = datetime.datetime.utcnow()
+    localized_datetime = pytz.timezone(tz).localize(utcnow)
+    return localized_datetime.strftime('%z')
+
+def format_timezone(tz):
+    """Format the given timezone to add the offset.
+
+    e.g.
+        'Africa/Cairo' -> '(UTC+02:00) Africa/Cairo'
+    """
+    offset = timezone_offset(tz)
+    return f'(UTC{offset[:3]}:{offset[3:]}) {tz}'
+
+_tzs = [
```

Comment:
This change impacts a lot of tz fields in Odoo, was not really in the specs... is it intended ?

Reply:
Yes, this was discussed with lna to be able to see the UCT difference and sort by timezone like Google Calendar does. Note that the current solution (tz, tz) doesn't really make much sense